### PR TITLE
Update to Elixir v1.9

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

File: posts/elixir/1/index.md [hunk 1]
```diff
@@ -617,19 +627,41 @@ iex(room1@COMPUTER-NAME)> Portal.Door.get(blue)
...
 
-  * Learn more about [ExUnit](http://elixir-lang.org/docs/stable/ex_unit/ExUnit.html), Elixir's testing framework, and write tests for the functionality we have built so far. Remember we already have a default structure laid out in the `test` directory.
+  * Learn more about [ExUnit](http://hexdocs.pm/ex_unit/ExUnit.html), Elixir's testing framework, and write tests for the functionality we have built so far. Remember we already have a default structure laid out in the `test` directory.
 
-  * Generate HTML documentation for your project with [ExDoc](http://github.com/elixir-lang/ex_doc).
+  * Generate HTML documentation for your project with [ExDoc](https://github.com/elixir-lang/ex_doc).
 
-  * Push your project to an external source, like [Github](https://github.com), and publish a package using the [Hex package manager](https://hex.pm).
+  * Push your project to an external source, like [GitHub](httpss://github.com), and publish a package using the [Hex package manager](https://hex.pm).
```
Comment (yurrriq) [inline posts/elixir/1/index.md]: `httpss` should be `https`

Comment (josevalim) [reply posts/elixir/1/index.md]: Guess who has a messed up macbook keyboard?

Thanks for the catch!

Comment (josevalim) [reply posts/elixir/1/index.md]:
```suggestion
  * Push your project to an external source, like [GitHub](https://github.com), and publish a package using the [Hex package manager](https://hex.pm).
```

File: posts/elixir/1/index.md [hunk 2]
```diff
@@ -330,9 +342,12 @@ def push_right(portal) do
 end
 ```
 
-We have defined our `Portal` struct and a `Portal.transfer/3` function (the `/3` indicates the function expects three arguments). Let's give this transfer a try. Start another shell with `iex -S mix` so our changes are compiled and type:
+We have defined a `Portal` struct and a `Portal.transfer/3` function (the `/3` indicates the function expects three arguments). Let's give this transfer a try. If IEx is still runinng, type `recompile()` to recompile the project (or run `iex -S mix` again):
```
Comment (tsloughter) [inline posts/elixir/1/index.md]:
```suggestion
We have defined a `Portal` struct and a `Portal.transfer/3` function (the `/3` indicates the function expects three arguments). Let's give this transfer a try. If IEx is still running, type `recompile()` to recompile the project (or run `iex -S mix` again):
```

**Pull request merged**